sirepo/package_data/static/js/sirepo.js

```diff
@@ -969,6 +970,18 @@ SIREPO.app.factory('notificationService', function(cookieService, $sce) {
     return self;
 });
 
+SIREPO.app.factory('simulationDataCache', function ($rootScope){
+    const self = {};
+    self.clear = () => {
+	for (let k in self) {
+	    if (k !== 'clear') {
+		delete self[k];
+	    }
+	}
+    };
+    return self;
+});
```

Comment:
No coding issues, but I don't quite get the difference between ```simulationDataCache``` and ```appState.applicationState()```, which seems to cache that data as well

Reply:
Does f74e1a6fa clarify?

Reply:
f74e1a6fa

Reply:
I see, thanks